### PR TITLE
Remove graduted command from experimental section docs

### PR DESCRIPTION
**Pull request by shamsher31**
To fix https://github.com/istio/istio.io/issues/7931

Based on https://github.com/istio/istio.io/issues/7931#issuecomment-674913422

The following are moved out of the experiment.
`analyze`  https://github.com/istio/istio/pull/19488
`convert-ingress` https://github.com/istio/istio/pull/16571
`dashboard` https://github.com/istio/istio/pull/17824, https://github.com/istio/istio/pull/26408
`upgrade` https://github.com/istio/istio/pull/19955

cc: @esnible Do suggest any other command that you think also needs to be hidden.

### CODE REVIEW

**Comment by shamsher31** [comment]
@esnible PTAL.

**Comment by shamsher31** [comment]
@esnible ping.

File: istioctl/cmd/root.go [hunk 1]
```diff
@@ -229,6 +225,24 @@ debug and diagnose their Istio mesh.
 	postInstallCmd.AddCommand(Webhook())
 	experimentalCmd.AddCommand(postInstallCmd)
 
+	analyzeCmd := Analyze()
+	hideInheritedFlags(analyzeCmd, "namespace", "istioNamespace")
```
Comment (esnible) [inline istioctl/cmd/root.go:229]: Why do we want to remove `--namespace`?  It still has a meaning for `analyze`.

For example, `istioctl analyze -n default` and `istioctl analyze -n bookinfo` are very different.

File: istioctl/cmd/root.go [hunk 2]
```diff
@@ -229,6 +225,24 @@ debug and diagnose their Istio mesh.
...
+	convertIngressCmd := convertIngress()
+	hideInheritedFlags(convertIngressCmd, "namespace", "istioNamespace")
+	rootCmd.AddCommand(convertIngressCmd)
+
+	dashboardCmd := dashboard()
+	hideInheritedFlags(dashboardCmd, "namespace", "istioNamespace")
+	rootCmd.AddCommand(dashboardCmd)
+
+	proxyStatusCmd := statusCommand()
+	hideInheritedFlags(proxyStatusCmd, "namespace", "istioNamespace")
```
Comment (esnible) [inline istioctl/cmd/root.go:241]: We need to keep `--namespace`.  It has no meaning for `istioctl proxy-status` but if the user does `istioctl proxy-status productpage-123` they can either use `-n bookinfo` or `productpage-123.bookinfo`.

**Comment by shamsher31** [comment]
> Thanks for working on these UX items. It would be great if you could attend UX meetings but I realize the time zone may be an issue. If you wish to attend maybe we can try some alternative time zones occasionally.

@esnible Sure, I will try to attend tomorrow, Usually happens at what time (IST)?

File: istioctl/cmd/root.go (hunk 1 above)
Comment (shamsher31) [reply istioctl/cmd/root.go]: Sorry, I misunderstood that part. Not sure if `istioNamespace` also needs to be kept? or if `hideInheritedFlags` is required for `analyze` ?

File: istioctl/cmd/root.go (hunk 2 above)
Comment (shamsher31) [reply istioctl/cmd/root.go]: Done

File: istioctl/cmd/root.go (hunk 1 above)
Comment (esnible) [reply istioctl/cmd/root.go]: We should keep `--istioNamespace` for analyze.

File: istioctl/cmd/root.go [hunk 3]
```diff
@@ -229,6 +225,24 @@ debug and diagnose their Istio mesh.
...
+
+	convertIngressCmd := convertIngress()
+	hideInheritedFlags(convertIngressCmd, "namespace", "istioNamespace")
+	rootCmd.AddCommand(convertIngressCmd)
+
+	dashboardCmd := dashboard()
+	hideInheritedFlags(dashboardCmd, "namespace", "istioNamespace")
+	rootCmd.AddCommand(dashboardCmd)
+
+	proxyStatusCmd := statusCommand()
```
Comment (esnible) [inline istioctl/cmd/root.go:240]: `proxy-status` uses the Istio Namespace to find control plane pods.  See _proxystatus.go_:85.

**Pull request merged**